### DISCUSSION TITLE
How to disable background-image: none !important

### DISCUSSION THREAD

**Discussion by ShahriarKh** [darkreader/darkreader#7100] [Q&A]
I'm using darkreader on my website. Everything is fine, but I don't want to remove the background image.
 what I have is something like this:
```
   background-color: orange;
   background-image: url("/images/patternwhite.png") !important;
   background-blend-mode: soft-light;
```
However, when I turn the dark mode on, this inline code is added that removes the pattern: `background-image: none !important`
Is there a way to change this behaviour?

**Comment by ShahriarKh** [comment]
One way I found is to put the background image inline instead of declaring it in an external CSS file. However, it somehow feels hacky and I'm still looking for an option to configure the darkreader, not changing my html/css.

**Comment by Gusted** [comment]
> ```
> 
> ```
> 
> 
>     
>   
> 
> However, when I turn the dark mode on, this inline code is added that removes the pattern: `background-image: none !important`
> Is there a way to change this behaviour?

That's pretty weird - is this a public site so I can have a look. It should only do that behavior in specific scenario's but have been thinking to remove it as it doesn't really gives us anythings good of it.

**Reply by ShahriarKh** [reply]
It's not public yet, but I will create a sample in codepen or codesandbox soon, so you can check it.

**Comment by deba-maity** [comment]
dude any solution. even putting url inline doesnt work

**Reply by Myshor** [reply]
Can you share some URL I could analyze how it is done and what can be (or not) suggested to fix it?
